package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "geotiff",
-  "version": "2.0.5",
+  "version": "2.0.6",
```

Comment:
Please don't update the version number. This is done automatically, when I issue the next release.

Reply:
Sounds good, I'll remove that commit.